com.unity.ml-agents/Documentation~/com.unity.ml-agents.md

```diff
@@ -124,3 +129,4 @@ Please refer to "Information that is passively collected by Unity" in the
 [execution order of event functions]: https://docs.unity3d.com/Manual/ExecutionOrder.html
 [connect with us]: https://github.com/Unity-Technologies/ml-agents#community-and-feedback
 [ml-agents forum]: https://forum.unity.com/forums/ml-agents.453/
+[ML-Agents GitHub repo]: https://github.com/Unity-Technologies/ml-agents/blob/release_12_docs/com.unity.ml-agents.extensions
```

Comment:
Should this be Release 13?

Reply:
@surfnerd Any idea why the pre-commit script missed this?

Reply:
Never mind, this was branched off the release branch before that was merged.